pkgs/applications/networking/instant-messengers/telegram/telegram-desktop/default.nix

```diff
@@ -198,6 +200,14 @@ stdenv.mkDerivation (finalAttrs: {
     ln -s $out/{Applications/${finalAttrs.meta.mainProgram}.app/Contents/MacOS,bin}
   '';
 
+  preFixup = ''
+    remove-references-to \
+      -t ${stdenv.cc.cc} \
+      -t ${microsoft-gsl} \
+      -t ${tg_owt.dev} \
```

Comment:
Strange, telegram-desktop doesn't seem to depend on these paths on current master.

Reply:
Indeed, looks like stdenv.cc.cc and tg_owt.dev have fixed on its own trough time. microsoft-gsl is likely just because it's now always bundled due to not having a release for a long time. But it seems it had a release a week ago so likely to start ti use the system one soon again... So it seems no need right now but may be needed in future for microsoft-gsl.